Sources/ViewInspector/SwiftUI/ViewThatFits.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+internal extension ViewType {
+    struct ViewThatFits {}
+}
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+extension ViewType.ViewThatFits: SingleViewContent {
+
+    static func child(_ content: Content) throws -> Content {
+        let view: Any = try {
+            guard let rootContent = try? Inspector.attribute(path: "_tree|content", value: content.view) else {
+                // A ViewThatFits View renders only one of its child Views based on the available horizontal space.
+                // This inspection returns all children that can potentially be picked by `ViewThatFits`.
+                // https://developer.apple.com/documentation/swiftui/viewthatfits
+                return try Inspector.attribute(path: "content", value: content.view)
+            }
+
+            return rootContent
+        }()
+        return try Inspector.unwrap(view: view, medium: content.medium)
```

Comment:
@nalexn can you help me better understand what it's appropriate to invoke this `unwrap(…)` method? I am trying to understand if we should always be unwrapping when extracting content from views.

Reply:
`unwrap` should be called when you get to a user-provided view that might be "wrapped" in modifiers. For example, `AnyView(x)`. This `x` can be a standard view, a custom user-defined view, or either of these but "wrapped" in a modifier, such as `x.padding(10)`. "unwrap" makes sure the root view of "x" is revealed for ease of further inspection (but modifiers are remembered as well, just stored separately).
When you inspect the internal child of a standard SwiftUI view, there is no need to "unwrap" them, if you can see from the code there is no generic parameter that can alter the view and turn it from `Text` into a `ModifiedContent<Text>`

Reply:
Thank you for the explanation!